meilisearch/client.py

```diff
@@ -996,3 +996,19 @@ def _valid_uuid(uuid: str) -> bool:
         )
         match = uuid4hex.match(uuid)
         return bool(match)
+
+    def get_experimental_features(self) -> dict:
+        """Get current experimental features settings."""
+        return self.http.get(self.config.paths.experimental_features)
+
+    def update_experimental_features(self, features: dict) -> dict:
+        """Update experimental features settings."""
+        return self.http.patch(self.config.paths.experimental_features, body=features)
+
+    def enable_multimodal(self) -> dict:
+        """Enable multimodal experimental feature."""
+        return self.update_experimental_features({"multimodal": True})
+
+    def disable_multimodal(self) -> dict:
+        """Disable multimodal experimental feature."""
+        return self.update_experimental_features({"multimodal": False})
```

Comment:
The `get_experimental_features` and `update_experimental_features` methods should be enough.

Those methods are unstable by nature because the features they can toggle change as the list of experimental features evolves.

If we add dedicated methods to enable/disable specific features, these too will be unstable. (We will need to remove them when the feature goes out of its experimental phase.)

It's preferable to have the minimum of unstable methods in the SDK.